plugin/init_npm.js

```diff
@@ -113,6 +97,6 @@ function _packageJsContent () {
   }
 
   Package.onUse(function(api) {
-    api.add_files(['index.js', '../../packages.json'], 'server');
```

Comment:
This is needed. Then meteor watches packages.json file the root of your app and reload when it changes.